net/instaweb/rewriter/rewrite_query.cc

```diff
@@ -545,7 +545,7 @@ RewriteQuery::Status RewriteQuery::ScanNameValue(
       if (pairs[i] == "no-transform") {
         // TODO(jmarantz): A .pagespeed resource should return un-optimized
         // content with "Cache-Control: no-transform".
-        options->set_enabled(RewriteOptions::kEnabledOff);
+        options->set_enabled(RewriteOptions::kEnabledStandby);
```

Comment:
I meant what happens if "off" or "unplugged" are query params... what I forgot is that IIRC we don't do that for .pagespeed. resources anyway?